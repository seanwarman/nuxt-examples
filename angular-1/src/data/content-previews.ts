import { RoomPlannerImages, SectionImages } from '../assets/images';

export interface ContentPreview {
  label: string;
  primaryImage: string;
  icon: string;
  body: string;
  slug: string;
  firstSubsection?: string;
  learnIcon: string;
  designIcon: string;
}

export const contentPreviews: ContentPreview[] = [
  {
    slug: 'camera-and-audio',
    label: 'Camera & Audio',
    firstSubsection: 'camera',
    primaryImage: SectionImages.cameraAndAudio.contentPreviewHero,
    icon: SectionImages.cameraAndAudio.icons.sectionIcon,
    designIcon: SectionImages.cameraAndAudio.icons.designIcon,
    learnIcon: SectionImages.cameraAndAudio.icons.learnIcon,
    body: `We've tuned and tweaked the hardware. Now, position it correctly to
     unlock the best performance.`,
  },
  {
    slug: 'displays-and-extras',
    label: 'Displays & Extras',
    firstSubsection: 'screen',
    primaryImage: SectionImages.displaysAndExtras.contentPreviewHero,
    icon: SectionImages.displaysAndExtras.icons.sectionIcon,
    designIcon: SectionImages.displaysAndExtras.icons.designIcon,
    learnIcon: SectionImages.displaysAndExtras.icons.learnIcon,
    body: `Correctly sizing displays is a key part of the conference experience.
      Plus, get some hints and tips on using the Series One Board 65 and Desk 27,
      developed in collaboration with Avocor,
      in your space.`,
  },
  {
    slug: 'room-layout',
    label: 'Room layout',
    firstSubsection: 'viewing-area',
    primaryImage: SectionImages.roomLayout.contentPreviewHero,
    icon: SectionImages.roomLayout.icons.sectionIcon,
    designIcon: SectionImages.roomLayout.icons.designIcon,
    learnIcon: SectionImages.roomLayout.icons.learnIcon,
    body: `Following a few spatial guidelines can make a huge difference for remote and in-room participants.`,
  },
  {
    slug: 'furniture-lighting',
    label: 'Furniture and lighting',
    firstSubsection: 'lighting',
    primaryImage: SectionImages.furnitureLighting.contentPreviewHero,
    icon: SectionImages.furnitureLighting.icons.sectionIcon,
    designIcon: SectionImages.furnitureLighting.icons.designIcon,
    learnIcon: SectionImages.furnitureLighting.icons.learnIcon,
    body: `Make your meeting room look and feel inviting with simple decorative and lighting tips. Appropriate lighting helps both local and remote participants have a great experience.`,
  },
  {
    slug: 'acoustics',
    label: 'Acoustics',
    firstSubsection: 'separation',
    primaryImage: SectionImages.acoustics.contentPreviewHero,
    icon: SectionImages.acoustics.icons.sectionIcon,
    designIcon: SectionImages.acoustics.icons.designIcon,
    learnIcon: SectionImages.acoustics.icons.learnIcon,
    body: `Poor acoustics is a common downfall of video conferencing spaces. The right allowance of acoustic finishes can help make speech clear and audible, and adequate isolation helps avoid interruptions.`,
  },
  {
    slug: 'services',
    label: 'Services',
    firstSubsection: 'air-handling',
    primaryImage: SectionImages.services.contentPreviewHero,
    icon: SectionImages.services.icons.sectionIcon,
    designIcon: SectionImages.services.icons.designIcon,
    learnIcon: SectionImages.services.icons.learnIcon,
    body: `Make sure power is provisioned for the right places, and ensure ventilation systems don't cause unwanted noise pickup on your microphones.`,
  },
  {
    slug: 'room-planner',
    label: 'Room Designer',
    primaryImage: RoomPlannerImages.contentPreviewHero,
    icon: RoomPlannerImages.icons.mainIcon,
    designIcon: SectionImages.services.icons.designIcon,
    learnIcon: SectionImages.services.icons.learnIcon,
    body: `Need to size up your room? All the best practices in this guide are used to help you pick the equipment and recommended room size for the occupancy you need.`,
  },
];
