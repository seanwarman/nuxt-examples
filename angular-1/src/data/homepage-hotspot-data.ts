export const HotspotData: Array<{
  name: string;
  text: string;
  color: string;
  defaultIcon: string;
  interimIcon: string;
  activeIcon: string;
  halo: string;
  x: number;
  y: number;
  z: number;
  cameraPosition: {
    x: number;
    y: number;
    z: number;
  };
  cameraRotation: {
    x: number;
    y: number;
    z: number;
  };
  modelsOn: Array<string>;
  modelsOff: Array<string>;
}> = [
  {
    name: 'camera-and-audio',
    text: 'Camera & Audio',
    color: '#e94233',
    defaultIcon: 'icon-hotspot-camera-audio-red-01-default@3x.png',
    interimIcon: 'icon-hotspot-camera-audio-red-01-interim@3x.png',
    activeIcon: 'icon-hotspot-camera-audio-red-01-active@3x.png',
    halo: 'icon-hotspot-camera-audio-red-01-active-halo@3x.png',
    x: -5,
    y: 113,
    z: -160,
    cameraPosition: {
      x: -100,
      y: 175,
      z: 350,
    },
    cameraRotation: {
      x: 100,
      y: 100,
      z: 0,
    },
    modelsOn: ['CameraAndAudio', 'defaultChairs', 'acousticDefault'],
    modelsOff: ['defaultCameraAndAudio'],
  },
  {
    name: 'displays-and-extras',
    text: 'Displays & Extras',
    color: '#0d9cab',
    defaultIcon: 'icon-hotspot-displays-extras-teal-01-default@3x.png',
    interimIcon: 'icon-hotspot-displays-extras-teal-01-interim@3x.png',
    activeIcon: 'icon-hotspot-displays-extras-teal-01-active@3x.png',
    halo: 'icon-hotspot-room-designer-teal-01-active-halo@3x.png',
    x: -230,
    y: 150,
    z: 50,
    cameraPosition: {
      x: 100,
      y: 200,
      z: 500,
    },
    cameraRotation: {
      x: 50,
      y: 100,
      z: 0,
    },
    modelsOn: [
      'DisplaysAndExtras',
      'defaultCameraAndAudioSetup',
      'defaultChairs',
      'defaultCameraAndAudio',
    ],
    modelsOff: [],
  },
  {
    name: 'room-layout',
    text: 'Room Layout',
    color: '#31a750',
    defaultIcon: 'icon-hotspot-room-layout-green-01-default@3x.png',
    interimIcon: 'icon-hotspot-room-layout-green-01-interim@3x.png',
    activeIcon: 'icon-hotspot-room-layout-green-01-active@3x.png',
    halo: 'icon-hotspot-room-layout-green-01-active-halo@3x.png',
    x: 90,
    y: 20,
    z: 200,
    cameraPosition: {
      x: 0,
      y: 280,
      z: 800,
    },
    cameraRotation: {
      x: 150,
      y: 100,
      z: 100,
    },
    modelsOn: [
      'RoomLayout',
      'defaultChairs',
      'defaultCameraAndAudioSetup',
      'acousticDefault',
      'defaultCameraAndAudio',
    ],
    modelsOff: [],
  },
  {
    name: 'furniture-lighting',
    text: 'Furniture & Lighting',
    color: '#fbbc05',
    defaultIcon: 'icon-hotspot-furniture-lighting-yellow-01-default@3x.png',
    interimIcon: 'icon-hotspot-furniture-lighting-yellow-01-interim@3x.png',
    activeIcon: 'icon-hotspot-furniture-lighting-yellow-01-active@3x.png',
    halo: 'icon-hotspot-furniture-lighting-yellow-01-active-halo@3x.png',
    x: 110,
    y: 70,
    z: 80,
    cameraPosition: {
      x: 400,
      y: 200,
      z: 150,
    },
    cameraRotation: {
      x: 0,
      y: 70,
      z: -100,
    },
    modelsOn: [
      'FurnitureAndLighting',
      'defaultCameraAndAudioSetup',
      'acousticDefault',
      'defaultCameraAndAudio',
    ],
    modelsOff: ['defaultChairs'],
  },
  {
    name: 'acoustics',
    text: 'Acoustics',
    color: '#3d85f2',
    defaultIcon: 'icon-hotspot-acoustics-blue-01-default@3x.png',
    interimIcon: 'icon-hotspot-acoustics-blue-01-interim@3x.png',
    activeIcon: 'icon-hotspot-acoustics-blue-01-active@3x.png',
    halo: 'icon-hotspot-acoustics-blue-01-active-halo@3x.png',
    x: -230,
    y: 200,
    z: 180,
    cameraPosition: {
      x: 300,
      y: 800,
      z: -150,
    },
    cameraRotation: {
      x: -150,
      y: 0,
      z: -100,
    },
    modelsOn: [
      'Acoustics',
      'defaultCameraAndAudioSetup',
      'defaultChairs',
      'acousticPanel00',
      'acousticPanel01',
      'acousticPanel02',
      'acousticPanel03',
      'defaultCameraAndAudio',
    ],
    modelsOff: ['stripLight00', 'stripLight01', 'acousticDefault'],
  },
  {
    name: 'services',
    text: 'Services',
    color: '#681da7',
    defaultIcon: 'icon-hotspot-services-purple-01-default@3x.png',
    interimIcon: 'icon-hotspot-services-purple-01-interim@3x.png',
    activeIcon: 'icon-hotspot-services-purple-01-active@3x.png',
    halo: 'icon-hotspot-services-purple-01-active-halo@3x.png',
    x: 110,
    y: 115,
    z: -160,
    cameraPosition: {
      x: 300,
      y: 160,
      z: 150,
    },
    cameraRotation: {
      x: 0,
      y: 50,
      z: -250,
    },
    modelsOn: [
      'services',
      'defaultCameraAndAudioSetup',
      'defaultChairs',
      'acousticDefault',
      'defaultCameraAndAudio',
    ],
    modelsOff: [],
  },
  {
    name: 'room-planner',
    text: 'Room Designer',
    color: '#0d9cab',
    defaultIcon: 'icon-hotspot-displays-extras-teal-01-default@3x.png',
    interimIcon: 'icon-hotspot-displays-extras-teal-01-interim@3x.png',
    activeIcon: 'icon-hotspot-room-designer-teal-01-active@3x.png',
    halo: 'icon-hotspot-room-designer-teal-01-active-halo@3x.png',
    x: -10,
    y: 90,
    z: 0,
    cameraPosition: {
      x: -30,
      y: 150,
      z: 150,
    },
    cameraRotation: {
      x: 40,
      y: 60,
      z: 0,
    },
    modelsOn: [
      'Drawing',
      'defaultCameraAndAudioSetup',
      'defaultChairs',
      'acousticDefault',
      'defaultCameraAndAudio',
    ],
    modelsOff: [],
  },
];
