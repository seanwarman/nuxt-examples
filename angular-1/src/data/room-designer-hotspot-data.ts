export const HotspotData: Array<{
  name: string;
  text: string;
  color: string;
  defaultIcon: string;
  interimIcon: string;
  activeIcon: string;
  halo: string;
  x: number;
  y: number;
  z: number;
  camX: number;
  camY: number;
  camZ: number;
}> = [
  {
    name: 'room-layout',
    text: 'Room Layout',
    color: '#31a750',
    defaultIcon: 'icon-hotspot-room-layout-green-01-default@3x.png',
    interimIcon: 'icon-hotspot-room-layout-green-01-interim@3x.png',
    activeIcon: 'icon-hotspot-room-layout-green-01-active@3x.png',
    halo: 'icon-hotspot-room-layout-green-01-active-halo@3x.png',
    x: -110,
    y: 20,
    z: 100,
    camX: -800,
    camY: 800,
    camZ: 800,
  },
  {
    name: 'camera-and-audio',
    text: 'Camera & Audio',
    color: '#e94233',
    defaultIcon: 'icon-hotspot-camera-audio-red-01-default@3x.png',
    interimIcon: 'icon-hotspot-camera-audio-red-01-interim@3x.png',
    activeIcon: 'icon-hotspot-camera-audio-red-01-active@3x.png',
    halo: 'icon-hotspot-camera-audio-red-01-active-halo@3x.png',
    x: -10,
    y: 90,
    z: 0,
    camX: -800,
    camY: 250,
    camZ: 300,
  },
  {
    name: 'displays-and-extras',
    text: 'Displays & Extras',
    color: '#0d9cab',
    defaultIcon: 'icon-hotspot-displays-extras-teal-01-default@3x.png',
    interimIcon: 'icon-hotspot-displays-extras-teal-01-interim@3x.png',
    activeIcon: 'icon-hotspot-displays-extras-teal-01-active@3x.png',
    halo: 'icon-hotspot-room-designer-teal-01-active-halo@3x.png',
    x: 0,
    y: 110,
    z: -160,
    camX: -1200,
    camY: 500,
    camZ: 600,
  },
  {
    name: 'services',
    text: 'Services',
    color: '#681da7',
    defaultIcon: 'icon-hotspot-services-purple-01-default@3x.png',
    interimIcon: 'icon-hotspot-services-purple-01-interim@3x.png',
    activeIcon: 'icon-hotspot-services-purple-01-active@3x.png',
    halo: 'icon-hotspot-services-purple-01-active-halo@3x.png',
    x: 120,
    y: 200,
    z: -160,
    camX: -120,
    camY: 250,
    camZ: 300,
  },
  {
    name: 'furniture-lighting',
    text: 'Furniture & Lighting',
    color: '#fbbc05',
    defaultIcon: 'icon-hotspot-furniture-lighting-yellow-01-default@3x.png',
    interimIcon: 'icon-hotspot-furniture-lighting-yellow-01-interim@3x.png',
    activeIcon: 'icon-hotspot-furniture-lighting-yellow-01-active@3x.png',
    halo: 'icon-hotspot-furniture-lighting-yellow-01-active-halo@3x.png',
    x: -70,
    y: 265,
    z: 20,
    camX: -120,
    camY: 300,
    camZ: 400,
  },
  {
    name: 'acoustics',
    text: 'Acoustics',
    color: '#3d85f2',
    defaultIcon: 'icon-hotspot-acoustics-blue-01-default@3x.png',
    interimIcon: 'icon-hotspot-acoustics-blue-01-interim@3x.png',
    activeIcon: 'icon-hotspot-acoustics-blue-01-active@3x.png',
    halo: 'icon-hotspot-acoustics-blue-01-active-halo@3x.png',
    x: -210,
    y: 190,
    z: -160,
    camX: -120,
    camY: 220,
    camZ: 580,
  },
  // {
  //   name: 'room-planner',
  //   text: 'Room Planner',
  //   color: '#0d9cab',
  //   defaultIcon: 'icon-hotspot-teal-01@3x.png',
  //   activeIcon: 'icon-room-designer-white-teal-circle-01@3x.png',
  //   halo: 'teal.png',
  //   x: -10,
  //   y: 150,
  //   z: -160,
  //   camX: 0,
  //   camY: 200,
  //   camZ: 200,
  // },
];
