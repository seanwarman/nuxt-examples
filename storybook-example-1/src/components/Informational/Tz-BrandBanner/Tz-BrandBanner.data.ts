export const brands = [
  {
    id: 8912,
    name: 'Altura',
    image: {
      src: '/brands/altura_logo.png',
      alt: 'Altura'
    },
    url: '/brand/altura'
  },
  {
    id: 3984,
    name: 'Boardman',
    image: {
      src: '/brands/boardman_logo.png',
      alt: 'Boardman'
    },
    url: '/brand/boardman'
  },
  {
    id: 9837,
    name: 'Cannondale',
    image: {
      src: '/brands/cannondale_logo.png',
      alt: 'Cannondale'
    },
    url: '/brand/cannondale'
  },
  {
    id: 4325,
    name: 'Cube',
    image: {
      src: '/brands/cube_logo.png',
      alt: 'Cube'
    },
    url: '/brand/cube'
  },
  {
    id: 5896,
    name: 'Endura',
    image: {
      src: '/brands/endura_logo.png',
      alt: 'Endura'
    },
    url: '/brand/endura'
  },
  {
    id: 7432,
    name: 'Giant',
    image: {
      src: '/brands/giant_logo.png',
      alt: 'Giant'
    },
    url: '/brand/giant'
  },
  {
    id: 4325,
    name: 'Cube',
    image: {
      src: '/brands/cube_logo.png',
      alt: 'Cube'
    },
    url: '/brand/cube'
  },
  {
    id: 5896,
    name: 'Endura',
    image: {
      src: '/brands/endura_logo.png',
      alt: 'Endura'
    },
    url: '/brand/endura'
  },
  {
    id: 7432,
    name: 'Giant',
    image: {
      src: '/brands/giant_logo.png',
      alt: 'Giant'
    },
    url: '/brand/giant'
  },
  {
    id: 6529,
    name: 'Merida',
    image: {
      src: '/brands/merida_logo.png',
      alt: 'Merida'
    },
    url: '/brand/merida'
  }
];
