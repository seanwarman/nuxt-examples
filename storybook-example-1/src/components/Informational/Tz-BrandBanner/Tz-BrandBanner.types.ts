import type { Image } from '../types';

export enum PageContext {
  Home = 'home',
  Hub = 'hub'
}

export interface Brand {
  id: number;
  name: string;
  image: Image;
  url: string;
}

export type BrandFilter = Omit<Brand, 'image' | 'url'>;

export interface Banner {
  page?: string;
  brands: Brand[];
}
