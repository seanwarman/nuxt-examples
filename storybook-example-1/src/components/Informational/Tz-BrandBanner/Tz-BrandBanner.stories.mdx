import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
import { action } from '@storybook/addon-actions';
import { withDesign } from 'storybook-addon-designs';

import TzBrandBanner from './Tz-BrandBanner.vue';
import { brands as brandData } from './Tz-BrandBanner.data';

<Meta
  title="Components/Informational/Brand Banner"
  component={TzBrandBanner}
  decorators={[withDesign]}
  argTypes={{
    page: { control: 'select', options: ['home', 'hub'] }
  }}
/>

export const Template = (args) => ({
  components: { TzBrandBanner },
  setup() {
    return { args };
  },
  template: `<tz-brand-banner v-bind="args"></tz-brand-banner>`
});

## Default Brand Banner

Allows a list of Brands supplied by the CMS to be rendered in a horizontally scrolled list.
The interaction of each Brand is described as a link as default, however when visiting a Hub page this acts as a filterable
callback.

<Canvas withSource="open">
  <Story
    name="Brands"
    args={{
      brands: brandData
    }}
    parameters={{
      design: {
        type: 'figma',
        url: 'https://www.figma.com/file/TU7n3pEHWIV5rWZUcUboOP/Ready-for-dev?node-id=1644%3A13692'
      }
    }}
  >
    {Template.bind({})}
  </Story>
</Canvas>

<ArgsTable story="Brands" />
