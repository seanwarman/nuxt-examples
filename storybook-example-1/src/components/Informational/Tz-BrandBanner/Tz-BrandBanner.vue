<template>
  <div class="brand-banner p-5 xl:container xl:px-5 xl:py-0">
    <tz-row cols="flow" :class="['gap-14', 'overflow-x-auto']">
      <tz-col :span="1" v-for="brand in brands" :key="brand.id">
        <div class="brand flex h-16 w-24 content-center justify-center">
          <a
            class="m-auto"
            :href="page == 'home' ? brand.url : undefined"
            @click="page == 'hub' ? filterBrandCallback($event, brand.id, brand.name) : null"
          >
            <img
              :class="['object-fit', 'hover:scale-100', 'scale-90', 'transition', 'duration-300']"
              :src="brand.image.src"
              :alt="brand.image.alt"
            />
          </a>
        </div>
      </tz-col>
    </tz-row>
  </div>
</template>

<script setup lang="ts">
import { defineEmits } from 'vue';
import TzRow from '@/components/Structural/Tz-Row/Tz-Row.vue';
import TzCol from '@/components/Structural/Tz-Col/Tz-Col.vue';

import type { Brand, BrandFilter } from './Tz-BrandBanner.types';
import { PageContext } from './Tz-BrandBanner.types';

export interface Banner {
  page?: string;
  brands: Brand[];
}

const emits = defineEmits<{
  (e: 'filterSelectedBrand', brand: BrandFilter): void;
}>();

const props = withDefaults(defineProps<Banner>(), {
  page: 'home'
});

/**
 * Callback for filtering
 * @param id: number
 *   The Brand identifier
 * @param name: string
 *   The name of the brand
 *
 * emits event "filterSelectedBrand"
 */
const filterBrandCallback = (e: Event, id: number, name: string) => {
  if (props.page == PageContext.Hub.toString()) {
    e.preventDefault();
  }
  emits('filterSelectedBrand', {
    id: id,
    name: name
  });
};
</script>

<style scoped>
/* Demonstrate a "mostly customized" scrollbar
 * (won't be visible otherwise if width/height is specified) */
::-webkit-scrollbar {
  width: 5px;
  height: 8px;
  background-color: #aaa; /* or add it to the track */
}

/* Add a thumb */
::-webkit-scrollbar-thumb {
  background: #000;
}
</style>
