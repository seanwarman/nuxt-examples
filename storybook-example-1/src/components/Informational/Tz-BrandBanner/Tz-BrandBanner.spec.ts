/**
 * @vitest-environment jsdom
 */

import { mount } from '@vue/test-utils';
import TzBrandBanner from './Tz-BrandBanner.vue';
import { expect, test } from 'vitest';
import { brands as brandsData } from './Tz-BrandBanner.data';
import type { Banner } from './Tz-BrandBanner.types';

const wrapper = mount(TzBrandBanner, {
  props: <Banner>{ page: 'home', brands: brandsData }
});

test('renders correctly', () => {
  expect(wrapper.element).toMatchSnapshot();
});

test('returns a brand id for the first brand value', () => {
  expect(wrapper.props().brands[0].id).toBeTypeOf('number');
});

test('behaves correctly on homepage', () => {
  if (wrapper.props().page == 'home') {
    expect(wrapper.find('a').attributes('href'));
  }
});
