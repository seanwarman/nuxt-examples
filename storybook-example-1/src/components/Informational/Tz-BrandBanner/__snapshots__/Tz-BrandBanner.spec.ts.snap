// Vitest Snapshot v1

exports[`renders correctly 1`] = `
<div
  class="brand-banner p-5 xl:container xl:px-5 xl:py-0"
  data-v-45b5b137=""
>
  <div
    class="tz-row grid auto-cols-max grid-flow-col gap-14 overflow-x-auto"
    data-v-45b5b137=""
  >
    
    
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/altura"
        >
          <img
            alt="Altura"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/altura_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/boardman"
        >
          <img
            alt="Boardman"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/boardman_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/cannondale"
        >
          <img
            alt="Cannondale"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/cannondale_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/cube"
        >
          <img
            alt="Cube"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/cube_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/endura"
        >
          <img
            alt="Endura"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/endura_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/giant"
        >
          <img
            alt="Giant"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/giant_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/cube"
        >
          <img
            alt="Cube"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/cube_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/endura"
        >
          <img
            alt="Endura"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/endura_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/giant"
        >
          <img
            alt="Giant"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/giant_logo.png"
          />
        </a>
      </div>
      
    </div>
    <div
      class="tz-col col-span-1"
      data-v-45b5b137=""
    >
      
      <div
        class="brand flex h-16 w-24 content-center justify-center"
        data-v-45b5b137=""
      >
        <a
          class="m-auto"
          data-v-45b5b137=""
          href="/brand/merida"
        >
          <img
            alt="Merida"
            class="object-fit hover:scale-100 scale-90 transition duration-300"
            data-v-45b5b137=""
            src="/brands/merida_logo.png"
          />
        </a>
      </div>
      
    </div>
    
    
  </div>
</div>
`;
